### ISSUE TITLE
Does not compile in Node.js v13

### ISSUE THREAD

**Issue by gajus** [DirtyHairy/node-libxl#25]
Logs:

```
> libxl@0.4.2 install /Users/gajus/Documents/dev/applaudience/applaudience-explorer-api/node_modules/libxl
> node install-libxl.js && node-gyp rebuild

Downloading http://www.libxl.com/download/libxl-mac-latest.tar.gz
successfully downloaded /var/folders/r5/wpcx3kt5681d1fzgtt2b1wsw0000gp/T/tmp-4418FpATKULSjqdglibxl-mac-latest.tar.gz, MD5: bba902af7fad7fcaefa2fd10c082840d
Extracting /var/folders/r5/wpcx3kt5681d1fzgtt2b1wsw0000gp/T/tmp-4418FpATKULSjqdglibxl-mac-latest.tar.gz ...
Renaming deps/libxl-mac-3.8.5.0 to deps/libxl ...
All done!
  CXX(target) Release/obj.target/liblibxl/src/bindings.o
  CXX(target) Release/obj.target/liblibxl/src/book.o
  CXX(target) Release/obj.target/liblibxl/src/argument_helper.o
../src/argument_helper.cc:76:41: error: no viable conversion from 'v8::Local<v8::Context>' to 'v8::Isolate *'
    return arguments[pos]->BooleanValue(Nan::GetCurrentContext()).ToChecked();
                                        ^~~~~~~~~~~~~~~~~~~~~~~~
/Users/gajus/Library/Caches/node-gyp/13.0.0/include/node/v8.h:2640:30: note: passing argument to parameter 'isolate' here
  bool BooleanValue(Isolate* isolate) const;
                             ^
1 error generated.
make: *** [Release/obj.target/liblibxl/src/argument_helper.o] Error 1
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/gajus/.nvm/versions/node/v13.0.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:210:5)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:272:12)
gyp ERR! System Darwin 19.0.0
gyp ERR! command "/Users/gajus/.nvm/versions/node/v13.0.0/bin/node" "/Users/gajus/.nvm/versions/node/v13.0.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Users/gajus/Documents/dev/applaudience/applaudience-explorer-api/node_modules/libxl
gyp ERR! node -v v13.0.0
gyp ERR! node-gyp -v v5.0.5
gyp ERR! not ok
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! libxl@0.4.2 install: `node install-libxl.js && node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the libxl@0.4.2 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/gajus/.npm/_logs/2019-10-24T19_22_10_663Z-debug.log

```

**Comment by DirtyHairy** [comment]
Fixed with 0.4.3. Sorry for the delay.

**Comment by gajus** [comment]
Thank you
